src/emulator/gui/src/game_selector.cpp

```diff
@@ -179,20 +181,61 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                     }
                     if (ImGui::IsMouseClicked(0))
                         selected[0] = true;
+                    if (gui.title_id != game.title_id)
+                        gui.title_id = game.title_id;
                 }
+
+                if (ImGui::BeginPopupContextItem("#game_context_menu")) {
+                    if (gui.title_id == game.title_id) {
+                        if (ImGui::MenuItem("Boot", game.title.c_str()))
+                            selected[0] = true;
+                        if (ImGui::BeginMenu("Copy Game Info")) {
+                            if (ImGui::MenuItem("ID and Name")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s, %s", gui.title_id, game.title.c_str());
+                                ImGui::LogFinish();
+                            }
+                            if (ImGui::MenuItem("ID")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s", gui.title_id);
+                                ImGui::LogFinish();
+                            }
+                            if (ImGui::MenuItem("Name")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s", game.title.c_str());
+                                ImGui::LogFinish();
+                            }
+                            ImGui::EndMenu();
+                        }
+                        if (ImGui::BeginMenu("Remove")) {
+                            if (ImGui::MenuItem("Game", "Need reboot after delete game")) { 
```

Comment:
Change to "Reboot is required after delete game". Why though, doesnt the list auto refresh

Reply:
Ok, for refresh i have founded nothings for now so if any dev how know refresh list ^^

Reply:
Yeah thats fair